pages/badges/badge-single.jsx

```diff
@@ -234,16 +234,26 @@ var BadgePage = React.createClass({
     );
   },
 
+  getShareCodes: function() {
+    var url = encodeURIComponent(window.location.toString());
+    var msg = encodeURIComponent(`I earned the Mozilla ${this.state.badge.title} badge`);
```

Comment:
either/or - from a user perspective I imagine they want to share a link that shows "them" as having earned a specific badge, but that would be a credly page (since we don't have pages for "a specific user's badge"). So alternatively we could at least make the share link point to the learning site page for that particular badge.